torch/_dynamo/variables/builder.py

```diff
@@ -777,7 +779,7 @@ def index_source(key):
             )
         elif isinstance(value, types.GetSetDescriptorType):
             self.install_guards(GuardBuilder.FUNCTION_MATCH)
-            return GetSetDescriptorVariable(value)
+            return GetSetDescriptorVariable(value, source=self.source)
```

Comment:
Are all the source changes still needed with set_source_and_track_mutable?  (Throughout the file)